source/expectations/ut_expectation_date.tps

```diff
@@ -16,6 +16,12 @@ create or replace type ut_expectation_date under ut_expectation(
   limitations under the License.
   */
   overriding member procedure to_equal(self in ut_expectation_date, a_expected date, a_nulls_are_equal boolean := null),
-  member procedure to_be_between(self in ut_expectation_date, a_lower_bound date, a_upper_bound date)
+  member procedure to_be_between(self in ut_expectation_date, a_lower_bound date, a_upper_bound date),
+  member procedure to_be_greater_or_equal(self in ut_expectation_date, a_expected date),
```

Comment:
Dont like the idea to add `not_to_be_less_than` cause its `to_be_greater_or_equal` etc.